.github/workflows/ci.yml

```diff
@@ -8,13 +8,9 @@ on:
   push:
     branches:
       - main
-    paths-ignore:
-      - "docs/**"
```

Comment:
The `docs` folder only contains markdown files with our site content. All of the actual "code" for the docs lives in `website`. At the end of the day, this doesn't matter much – but is there a reason for removing this `docs/**` excludes so that CI runs on docs-only changes? (I might be missing something, are we running prettier on the docs MD files now?)

Reply:
This is an artifact of the old site where docs had all the content and code. At any rate, since we do quality checks (lint, prettier) in this workflow for everything now in this PR we shouldn't have `paths-ignore`